### ISSUE TITLE
Changing fullscreen doesn't work

### ISSUE THREAD

**Issue by maxklock** [Acagamics/virusx#16]
When changing from fullscreen to window, it's always changing back to fullscreen

**Comment by Wumpf** [comment]
Is this a UWP only issue? In UWP going to fullscreen is only a request, so it may fail. Is what the option screen shows at least correct?
I can probably need to add some logging on the issue, it is hard to tell otherwise what is going on. Worked always fine on my computer.

**Comment by maxklock** [comment]
I'm using the "Fullscreen"-Button in the titlebar.

**Comment by Wumpf** [comment]
... there is a fullscreen bar in the titlebar?! Isn't that what #12 is asking for? Or are you referring to Maximize?

**Comment by maxklock** [comment]
![2016-08-23](https://cloud.githubusercontent.com/assets/7167301/17891181/837a321c-693a-11e6-9c12-b4108b0aee92.png)

There is an Fullscreen-Button instead of maximize

**Comment by Wumpf** [comment]
Yeah I think that is what Gerd wants in #12. I don't have that and apparently neither does he. Which Windows build are you using?

**Comment by maxklock** [comment]
No, because when using the Options to Change to window, I don't have the button. It's an automatic added Button to Change to window, when in Fullscreen. But #12 is the other direction
